src/main/msp/msp.c

```diff
@@ -3002,6 +3003,17 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
         }
         break;
 
+    case MSP2_SEND_DSHOT_COMMAND:
+        {
+#ifdef USE_DSHOT
```

Comment:
The conditional should go around the entire `case` - this way, if firmware has been compiled without the feature, MSP will indicate this with an error.